.gitignore

```diff
@@ -0,0 +1,399 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
ignore `reports` folder and all files inside

Reply:
Not done yet

Reply:
AGAIN, this is not done. You have to ignore the \\reports\* in gitignore

Reply:
Done, I added it in line 33
![image](https://user-images.githubusercontent.com/79431746/204850459-1a1ed553-3f4c-466e-b95b-ca73afd562e1.png)

Reply:
Remove this line

Reply:
Done